definition/wof_controlled.md

```diff
@@ -0,0 +1,28 @@
+## What is the wof:controlled property used for?
+
+Many Who’s On First records have a `wof:controlled` property, a top-level property used to "freeze" other property values in a given Who’s On First record. The `wof:controlled` property is a list that can contain any other existing property keys in a Who’s On First record. The most common use of this property is to "freeze" the `wof:hierarchy` and `wof:parent_id` properties, as follows:
+
+```
+"wof:controlled":[
+    "wof:hierarchy",
+    "wof:parent_id"
+]
+```
+
+In the case above, the `wof:controlled` property includes `wof:hierarchy` and `wof:parent_id`; this results in no future updates to this record's `wof:hierarchy` or `wof:parent_id` values.
```

Comment:
In the case above, the `wof:controlled` property includes `wof:hierarchy` and `wof:parent_id`; because Who's On First tools respect controlled properties during automatic updates any potential changes to the records `wof:hierarchy` or `wof:parent_id` values would be skipped.